prisma/schema.prisma

```diff
@@ -187,6 +187,8 @@ model Exercise {
   answer      String
   testStr     String?
   explanation String?
+  flagged     Boolean?
```

Comment:
instead of making flagged optional thereby giving it essentially 3 possible values (null, false, true) would be better to make it required with a default value of false
e.g. `flagged   Boolean @default(false)`

Alternatively, make the field flagged_at, optional and of the type date, where null means it is not flagged, and a timestamp means it is flagged and the timestamp is the time it was flagged at.
e.g. `flagged_at DateTime?`

Reply:
`flagged_at` is good suggestion thank you

Reply:
Sounds good!